src/main/java/mate/academy/internetshop/service/impl/BucketServiceImpl.java

```diff
@@ -73,6 +73,14 @@ public void clear(Bucket bucket) {
         bucketDao.update(tempBucket);
     }
 
+    @Override
+    public Bucket getByUserId(Long userId) {
+        return bucketDao.getAll().stream()
+                .filter(x -> x.getUserId().equals(userId))
+                .findFirst()
+                .orElse(create(new Bucket()));
```

Comment:
```suggestion
                .orElse(create(new Bucket(userId)));
```
якщо так не зробити, то твій Бакет буде лежати в "БД" із userId=null